yamllint/rules/quoted_strings.py

```diff
@@ -139,6 +166,8 @@ def VALIDATE(conf):
         return 'cannot use both "required: true" and "extra-required"'
     if conf['required'] is False and len(conf['extra-allowed']) > 0:
         return 'cannot use both "required: false" and "extra-allowed"'
+    if conf['skip-quoted-quote'] is True and conf['quote-type'] == 'any':
+        return '"skip-quoted-quote" has no effect with "quote-type: any"'
```

Comment:
I fear it can be unfriendly to the user to fail on that. Maybe we can just silently ignore `skip-quoted-quote` in that case, and run yamllint normally?

Reply:
Sounds reasonable. I've removed this check.